internal/example/README.md

```diff
@@ -0,0 +1,16 @@
+# Example xDS Server
+
+This is an example of a trivial xDS V3 control plane server.  It serves an Envoy configuration that's roughly equivalent to the one used by the Envoy ["Quick Start"](https://www.envoyproxy.io/docs/envoy/latest/start/start#quick-start-to-run-simple-example) docs: a simple http proxy.  You can run the example using the project top-level Makefile, e.g.:
+
+```
+go-control-plane$ make example
+```
+
+The Makefile builds the example server and then runs `build/example.sh` which runs both Envoy and the example server.  The example server serves a configuration defined in `internal/example/resource.go`.  If everything works correctly, you should be able to open a browser to [http://localhost:10000](http://localhost:10000) and see Acnodal.io's website.
```

Comment:
Do you think we can use something neutral, e.g. envoyproxy.io?

Reply:
Yup, I pushed a fixup commit to change it to www.envoyproxy.io.  LMK if you'd like me to squash the fixup commits and force push.